web/js/index.js

```diff
@@ -101,9 +109,10 @@ const initThreeJsElements = function() {
             camera.far = size * 100;
             camera.updateProjectionMatrix();
             camera.position.copy(center);
-            camera.position.x += size / 2.0;
-            camera.position.y += size / 5.0;
-            camera.position.z += size / 2.0;
+            const boxSize = box.getSize();
+            camera.position.x = boxSize.x * 2;
+            camera.position.y = boxSize.y * 2;
+            camera.position.z = boxSize.z * 2;
```

Comment:
I'm trying to reason about this. I think this new position means that if looking at something tall and narrow, you'll look down on it - essentially the camera would always go closer to the object's major axis, which I don't think is what Onshape itself does. Might be good to ask Evan Nowak (he is on slack) about what to do here. @evannowak

Reply:
It _seems_ to work OK. But I agree, it's probably not ideal.